RuddockWebsite/modules/account/templates/create_account.html

```diff
@@ -43,8 +43,8 @@
       <td>{{ user_data['email'] }}</td>
     </tr>
     <tr>
-      <td>Birthday (YYYY-MM-DD)</td>
-      <td><input type="text" name="birthday" /></td>
+      <td>Birthday</td>
+      <td><input type="text" name="birthday" placeholder="YYYY-MM-DD" /></td>
```

Comment:
Not a huge fan of making day/month/year separate inputs, since then they have to be re-concatenated on the server to be used. I do agree that it's more visually appealing, though. I wish the date input was implemented on more browsers :-\

I also like using the ISO standard for dates:
https://xkcd.com/1179/

:p